src/Raven.Studio/typescript/common/rqlLanguageService.ts

```diff
@@ -16,7 +17,7 @@ class rqlLanguageService {
     
     constructor(
         db: database, 
-        indexes: KnockoutObservable<string[]>,
+        indexes: KnockoutObservable<string[]> | string[],
```

Comment:
what if we put here: `indexes: ()=> string[]` ?

Reply:
which is "provider" of index names.

Reply:
All right